src/main/jhipster/src/main/webapp/app/shared/variable-select/variable-select.component.ts

```diff
@@ -13,8 +14,17 @@ declare const $: any;
     templateUrl: './variable-select.component.html'
 })
 export class VariableSelectComponent implements OnInit {
-    @Input() multiple: boolean;
-    @Input() variableTypeIds: string[];
+    @Input() name: string;
+    @Input() id: string;
+
+    @Input() value: any;
+    initialData: Select2OptionData[];
+
+    @Input() disabled = false;
+    @Input() multiple = false;
+    @Input() variableTypeIds: number[];
```

Comment:
Can `variableTypeIds` here be declared as `VariableType[]` (noting that VariableType here was the renaming of VariableTypeEnum)